### PR TITLE
improve recipe mappings

### PR DESCRIPTION
**Pull request by eristhea**
a few mappings requested by the QSL gang

### CODE REVIEW

File: mappings/net/minecraft/recipe/RecipeGridAligner.mapping [hunk 1]
```diff
@@ -6,3 +6,9 @@ CLASS net/minecraft/unmapped/C_wuyfqgmj net/minecraft/recipe/RecipeGridAligner
 		ARG 4 recipe
 		ARG 5 inputs
 		ARG 6 amount
+	METHOD m_vnshednj acceptAlignedInput (Ljava/lang/Object;IIII)V
+		ARG 1 input
+		ARG 2 index
+		ARG 3 amount
+		ARG 4 y
+		ARG 5 x
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/recipe/RecipeGridAligner.mapping]:
```suggestion
		ARG 4 x
		ARG 5 y
```
Swapped. Also noticed `RecipeAlternativesWidget.InputSlot`'s `x` and `y` are swapped when I was checking this.

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/recipe/RecipeGridAligner.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/recipe/RecipeGridAligner.mapping]: that's where I got the names, I'll fix this

Comment (eristhea) [reply mappings/net/minecraft/recipe/RecipeGridAligner.mapping]: done!

**Review by supersaiyansubtlety — APPROVED** [review]

**Comment by supersaiyansubtlety** [comment]
Is this supposed to still have the 'wip' label?

**Comment by eristhea** [comment]
no it is not!
that would explain the lack of reviews lol

**Comment by eristhea** [comment]
superceded by #593

**Pull request closed without merging**